nixos/modules/services/networking/bitcoind.nix

```
@@ -59,7 +59,7 @@ in {
 
       package = mkOption {
         type = types.package;
-        default = pkgs.altcoins.bitcoind;
+        default = pkgs.bitcoind;
         defaultText = "pkgs.altcoins.bitcoind";
```

This should be the last reference
```suggestion
        defaultText = "pkgs.bitcoind";
```